clusters/core/addons/argo-cd/templates/oidc/keycloak-group-krci.yaml

```diff
@@ -0,0 +1,11 @@
+{{ if .Values.oidc.enabled }}
+apiVersion: v1.edp.epam.com/v1
+kind: KeycloakRealmGroup
+metadata:
+  name: krci-users
```

Comment:
do we need to parametrize here? or let's at least add commend that this role is created on KRCI side and name must be aligned